draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -848,6 +848,152 @@ Content-Type: text/plain
+   also be provided by other protocols outside the scope of this document.
+</t>
+<t>
+   The "https" scheme, in contrast, associates authority based on the ability
+   of a server to use the private key associated with a certificate that
+   the client considers to be trustworthy for the identified host.
+   If a server presents a certificate that verifiably applies to the host,
+   along with proof that it controls the private key, then a client will
+   accept a secured connection to that server as being authoritative for all
+   origins with the same host. However, each origin remains distinct (from
+   origins with a different scheme or port) unless explicitly aliased by an
+   origin server verified to be under the same authority.
+</t>
+<t>
+   What the client is relying on for "https" authority is that there exists
+   some chain of trust (possibly configured by exception) that verifies the
+   certificate provided. Hence, the authority is defined only by the host,
+   the port impacts which certificate is chosen (assuming that certificate
+   matches the host identifier), and the client trusts that match according
+   to the secure handshake and certificate validation described in
+   <xref target="https.initiation"/>.
+</t>
+<t>
+   Note that the "https" scheme does not rely on TCP and the port number for
+   establishing authority, even though it provides for use of HTTP over TLS
+   over TCP via that port as the default mechanism for obtaining a
+   certificate that can be used for verifying authority and providing access
+   to the origin's resources. The peers establishing a secure connection
+   cannot trust transport parameters outside the secured channel, but the
+   port number remains useful (beyond the default mechanism) as an identifier
+   to distinguish different origin namespaces on the same host. The port
+   identifier is therefore applicable even when the protocols being used for
+   access do not include TCP.
+</t>
+<t>
+   In HTTP/1.1 and earlier, the only URIs for which the client will assign
+   authority are those that contain a TLS server_name extension matching the
+   origin's host.
+</t>
+<t>
+   In HTTP/2, the client will assign authority to all names that are present
+   in the certificate. However, a client will only do that if it concludes
+   that it could open a connection to the origin for that URL. In practice, a
+   client will make a DNS query and see that it contains the same server IP
+   address. A server sending the ORIGIN frame removes this restriction
+   <xref target="RFC8336"/>.
+</t>
+<t>
+   In addition to the client's verification, the origin server is responsible
+   for verifying that that requests it receives over a connection correspond
+   to resources for which it actually wants to be the origin. If a network
+   attacker causes connections for port N to be received at port Q, checking
+   the Host header field is necessary to ensure that the attacker can't cause
```

Comment:
..., the `:authority` pseudo-header field, (or the port field from a request in absolute form?).

h2 recommends use of `:authority` over `Host`.

Reply:
Changed to "effective request URI"